coalib/bears/Bear.py

```diff
@@ -9,6 +9,12 @@
 from coalib.settings.ConfigurationGathering import get_config_directory
 
 
+class classproperty(property):
+
+    def __get__(self, obj, type_):
+        return self.fget.__get__(None, type_)()
```

Comment:
I think you can do directly `classmethod(self.fget). ...`, then you can declare a classproperty without declaring it as `classmethod` explicitly.
But you can google classproperties in python there're enough snippets out there^^